drivers/sx127x/include/sx127x_netdev.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2017 Inria
+ *               2017 Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_sx127x
+ * @{
+ * @file
+ * @brief       Netdev driver definitions for SX127X driver
+ *
+ * @author      José Ignacio Alamos <jose.alamos@inria.cl>
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef SX127X_NETDEV_H
+#define SX127X_NETDEV_H
+
+#include "net/netdev.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Reference to the netdev device driver struct
+ */
+extern const netdev_driver_t sx127x_driver;
+
+/**
+ * @brief   Received LoRa packet status information
+ */
+typedef struct netdev_radio_lora_packet_info {
+    uint8_t rssi;           /**< RSSI of a received packet */
+    uint8_t lqi;            /**< LQI of a received packet */
```

Comment:
I just had a look at this code again due to an unrelated issue I discussed with @kYc0o, so sorry for the "late review" ;-).

In the [`recv()` function of this driver](https://github.com/RIOT-OS/RIOT/pull/6797/files#diff-37ee9c4f2a20ec7a38d429c04255744aR155) it says that LoRA has no LQI, so why is it in this struct?

Reply:
indeed it shouldn't be here. I will fix it, since I require it for #11022